code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -182,3 +182,7 @@
 
 /mob/living/simple_animal/drone/cogscarab/ratvar_act()
 	fully_heal(TRUE)
+
+/obj/item/drone_shell/dusty
+	name = "dust-caked drone shell"
```

Comment:
Correction: I am **more** excited about golems than I am drones. It is just that I do not like getting them mixed up!